### PR TITLE
Add 'Flattr AB' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit in company JSON generator](https://company-json.datenanfragen.de/#!url=https%3A%2F%2Fraw.githubusercontent.com%2Fdatenanfragen-community-edits%2Fdata%2Fsuggest_flattr-eyeo_1696454967778%2Fcompanies%2Fflattr-eyeo.json )**

### CODE REVIEW

**Comment by rugk** [comment]
My PR (would love it if you could signup with your GitHub account o have it associated [like on webcompat.com, which seems to work similar)… BTW. (Or shall I open a new issue for that?)

File: companies/flattr-eyeo.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "https://flattr.com/privacy",
+        "https://github.com/datenanfragen/data/pull/2380"
+    ],
+    "required-elements": [
+        {
+            "desc": "E-Mail",
+            "type": "email",
+            "optional": false
+        }
+    ],
```
[the remark this answers is not in the document]
Comment (rugk) [reply companies/flattr-eyeo.json:23]: I did not see how to enter that in the web form, it had a "Name:" that appeared here as a "desc".

**Comment by baltpeter** [comment]
> My PR (would love it if you could signup with your GitHub account o have it associated [like on [webcompat.com](https://webcompat.com), which seems to work similar)… BTW. (Or shall I open a new issue for that?)

I am actually currently working on something just like that. It's currently in a pre-alpha state though and it will take some time until I can properly release it.

File: companies/flattr-eyeo.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (baltpeter) [reply companies/flattr-eyeo.json:23]: Ah sorry, the wording maybe isn't clear enough. This actually doesn't refer to a property `name` on a required element but rather to one required element of `type: name`.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**